chronoline/chronoline.js

```diff
@@ -86,7 +86,7 @@ function getLeft(elem){
 }
 
 function getEndDate(dateArray){
-    return dateArray[dateArray.length - 1];
+    return new Date(Date.parse(dateArray[dateArray.length - 1]));
```

Comment:
what was the motivation for this? I legitimately can't remember why the code looks the way it does in the first place to say otherwise, but just want some context on this